### DISCUSSION TITLE
duplicate key value violates unique constraint "netbox_secrets_secret_pkey"

### DISCUSSION THREAD

**Discussion by KRONNESAP** [Onemind-Services-LLC/netbox-secrets#82] [Q&A]
I'm using the netbox v3.5.5 version and the netbox-secrets v1.8.5 plugin.
I can decrypt passwords normally, but when trying to enter a new password, this error appears below:

![screenshot_05](https://github.com/Onemind-Services-LLC/netbox-secrets/assets/106918711/6df97dc6-c4fe-4e13-b76d-4439d10cf5ad)

Does anyone know the solution to this problem?

**Comment by abhi1693** [comment]
If you migrated from netbox-secretstore, then this problem is hihly likely to arrive if you do not complete the steps provided in the FAQ for data migration. Steps specifically for fixing the sequence IDs in the database.

**Reply by KRONNESAP** [reply]
This is the command: **python manage.py sqlsequencereset netbox_secrets**. It ran successfully during the migration, yes. Is there anything I can do to fix it directly at the bank if that's the case?

**Reply by abhi1693** [reply]
That only generates the commands that you need to run yourself inside your DB.

**Reply by KRONNESAP** [reply]
Exactly. It's the last thing to do when doing the migration process as per the guidance at the bottom of this page: https://github.com/Onemind-Services-LLC/netbox-secrets

Even with everything running without any error occurring. This problem happens.

**Reply by abhi1693** [reply]
I looked into this and on my setup the only way this occurred was when I did not reset the DB sequences because the migrations causes inconsistencies. So, unless there are more clear steps to reproduce this as a bug, I'd suggest to look into the steps that you have run because this seems like a user error at the moment.
